src/ResultWriter/EnergyOutput.cpp

```diff
@@ -700,22 +751,42 @@ void EnergyOutput::checkAbortCriterion(real timeSinceThreshold, const std::strin
 void EnergyOutput::writeHeader() { out << "time,variable,measurement" << std::endl; }
 
 void EnergyOutput::writeEnergies(double time) {
-  if (shouldComputeVolumeEnergies()) {
-    out << time << ",gravitational_energy," << energiesStorage.gravitationalEnergy() << "\n"
-        << time << ",acoustic_energy," << energiesStorage.acousticEnergy() << "\n"
-        << time << ",acoustic_kinetic_energy," << energiesStorage.acousticKineticEnergy() << "\n"
-        << time << ",elastic_energy," << energiesStorage.elasticEnergy() << "\n"
-        << time << ",elastic_kinetic_energy," << energiesStorage.elasticKineticEnergy() << "\n"
-        << time << ",momentumX," << energiesStorage.totalMomentumX() << "\n"
-        << time << ",momentumY," << energiesStorage.totalMomentumY() << "\n"
-        << time << ",momentumZ," << energiesStorage.totalMomentumZ() << "\n"
-        << time << ",plastic_moment," << energiesStorage.plasticMoment() << "\n";
+  for (size_t sim = 0; sim < multipleSimulations::numberOfSimulations; sim++) {
+#ifdef MULTIPLE_SIMULATIONS
```

Comment:
You could e.g. introduce a `constexpr` with `MultipleSimulations == true/false`. Then you can also shorten the string initialization here.